resolwe_bio/tools/cuffnorm_output_table_headers.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Rename cuffnorm output tables headers."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+
+parser = argparse.ArgumentParser(description="Rename cuffnorm output tables headers")
+parser.add_argument('sample_names', help="Sample names")
+args = parser.parse_args()
+
+sample_names=[]
+
+for sample_file in args.sample_names.split(","):
```

Comment:
How is `sample_file`connected with `sample_name`?

Sample name is unrelated to any file name, so i think that `sample_file` is not the best choice for this variable.